project/plugins.sbt

```diff
@@ -1,11 +1,11 @@
 logLevel := Level.Warn
 
-resolvers += "bintray-spark-packages" at "https://dl.bintray.com/spark-packages/maven/"
+resolvers += Resolver.bintrayIvyRepo("s22s", "sbt-plugins")
 
-addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.6")
+addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.7-astraea.1")
```

Comment:
Maybe we need an official release of sbt-spark-package?

Otherwise, we are getting more technical debt here.